File: wait_formula/Wait_formula.py
from sympy import *

r, eta, beta, rho, a, h = symbols('r eta beta rho a h', real=True)

beta = 2*pi/100
a = 18
h = 20
r = sqrt(rho**2 + h**2)

expr0 = (-eta)/((2*pi*sin(beta*h)**2))
expr1 = Integral(exp(-1j*2*beta*r)/rho, (rho, a, oo)) 
expr2 = -1*2*cos(beta*h)*Integral(exp(-beta*1j*(r + rho))/rho, (rho, a, oo)) 
expr3 = -cos(beta*h)**2*Integral(exp(-1j*2*beta*rho)/rho, (rho, a, oo))

norm = 4*pi*expr0/eta

norm_deltaZ_re = re(norm.evalf() * (expr1.evalf() + expr2.evalf() + expr3.evalf()))

gwave_Rr = norm_deltaZ_re.evalf()
print(gwave_Rr)

File: wait_formula/Wait_formula.m
syms r eta beta rho a h real

beta = 2*pi/100;
a = 18;
h = 20;

r = sqrt(rho^2 + h^2);

expr0 = (-eta)/((2*pi*sin(beta*h)^2));
expr1 = int(exp(-1i*2*beta*r)/rho, rho, a, inf); 
expr2 = -1*2*cos(beta*h)*int(exp(-beta*1i*(r + rho))/rho, rho, a, inf); 
expr3 = -cos(beta*h)^2*int(exp(-1i*2*beta*rho)/rho, rho, a, inf);

norm = 4*pi*expr0/eta;

norm_deltaZ_re = real(vpa(norm) * (vpa(expr1) + vpa(expr2) + vpa(expr3)))
